api/pkg/transformer/feast/validator.go

```diff
@@ -52,8 +52,11 @@ func ValidateTransformerConfig(ctx context.Context, coreClient core.CoreServiceC
 				return NewValidationError("entity not found: " + entity.Name)
 			}
 
-			if len(entity.JsonPath) == 0 {
-				return NewValidationError(fmt.Sprintf("json path for %s is not specified", entity.Name))
+			switch entity.Extractor.(type) {
```

Comment:
Can you add validation logic for the udf config provided by user?

Reply:
Is it suffice to verify if the string is non empty, or do we need to make sure that it can be compiled without error as well? If the latter, right now initializing new transformer would have thrown error if the udf expression can't be compiled, so not sure whether we need an additional verification.

Reply:
It's better to call `Compile` too. These validation steps are performed when user configure the standard transformer from the UI. If we chose the later, the error only show up as deployment failure and might confuse user. I think we forgot to add the validation for `jsonPath` too, so it will be great if you can add it here.